less/index.less

```diff
@@ -57,3 +57,68 @@ html, body {
     height: 100%;
 }
 
+/* My CSS Code Starts Here */
+
+body{
+	font: 15px/1.5 Arial, Helvetica, sans-serif ;
+	max-width: 1000px;
+	padding:0;
```

Comment:
No need to indicate `padding: 0` since you aren't selecting padding on this anywhere else